### PR TITLE
DEV: Don't lint core files when target == plugins

### PR DESCRIPTION
**Pull request by CvX**
Less work for the GitHub Actions and less noise in the results.

### CODE REVIEW

**Comment by tgxworld** [comment]
Looks good to me :) feel free to merge

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -136,22 +136,36 @@ jobs:
...
+        if: env.BUILD_TYPE == 'LINT' && env.TARGET == 'CORE'
         run: |
           yarn prettier -v
           yarn prettier --list-different "app/assets/stylesheets/**/*.scss" "app/assets/javascripts/**/*.js" "app/assets/javascripts/**/*.es6" "test/javascripts/**/*.es6" "plugins/**/*.scss" "plugins/**/*.es6"
 
+      - name: Prettier (all plugins)
+        if: env.BUILD_TYPE == 'LINT' && env.TARGET == 'PLUGINS'
+        run: |
+          yarn prettier -v
+          yarn prettier --list-different "plugins/**/*.scss" "plugins/**/*.es6"
```
Comment (davidtaylorhq) [inline .github/workflows/ci.yml:167]: Do we need to add `.js` here, since we have stopped using `.es6` in some plugins?

Comment (CvX) [reply .github/workflows/ci.yml:167]: Ha, good eye! So we were not linting some files then. 😅

Comment (davidtaylorhq) [reply .github/workflows/ci.yml:167]: Yeah I guess so!

Sorry to bring it up in this PR, it's not really related. Maybe we should fix that separately?

Comment (CvX) [reply .github/workflows/ci.yml]: Too late, added to this PR 😜

Comment (davidtaylorhq) [reply .github/workflows/ci.yml]: Looks like we were indeed missing some linting failures 😅

Comment (CvX) [reply .github/workflows/ci.yml]: Yup, https://github.com/discourse/discourse/pull/10260 😁

**Comment by SamSaffron** [comment]
looks like we have general approval here @CvX  can you merge when you start your day ?

**Comment by tgxworld** [comment]
@CvX Is this good to merge?

**Comment by CvX** [comment]
Now it is 😃

**Comment by tgxworld** [comment]
Merge away :)

**Pull request merged**